docs/05_components/doc-management/doc-management-server/01_server-configuration.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: 'Document Management - Initial Configuration'
+sidebar_label: 'Initial Configuration'
+id: doc-management-initial-configuration
+keywords: [doc-management, server-configuration]
+tags:
+  - doc-management
+  - server-configuration
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Initialising & Adding to a New Application
+
+The Document Management component can be added to a new application using GenX - see [GenX CLI](/docs/06_operations/03_commands/04_Genx_CLI.mdx) for more details.
+
+The following command will initialise a new application called `myapp` and add the Document Management component to it.
+
+<Tabs
```

Comment:
I'm wondering if the genx install commands should be one level above rather than nested in the server directory, as it installs both client and server

Reply:
Was thinking the same - will collab with Zuhaa, I believe she has done the same. Will unbox and put into one of our branchs